src/components/sequencing/SequenceEditor.svelte

```diff
@@ -99,18 +96,12 @@
     ? userSequenceEditorColumnsWithFormBuilder
     : userSequenceEditorColumns;
 
-  $: librarySequenceMap = Object.fromEntries(librarySequences.map(seq => [seq.name, seq]));
-  $: phoenixContext = {
-    channelDictionary,
-    commandDictionary,
-    parameterDictionaries,
-    librarySequenceMap,
-  };
   $: {
-    if (!commandDictionary) {
-      commandDictionary = null;
-      channelDictionary = null;
-      parameterDictionaries = [];
+    // TODO why is this necessary? Can we not just handle it all together??
```

Comment:
This is suppose to be trigger when the user selects a different parcel.  We can't change the parcel anymore at the sequence editor level so I think we can remove it.